hal/src/thumbv6m/clock/v1.rs

```diff
@@ -521,10 +523,13 @@ pub fn enable_internal_32kosc(sysctrl: &mut SYSCTRL) {
 /// Turn on the external 32hkz oscillator
 pub fn enable_external_32kosc(sysctrl: &mut SYSCTRL) {
     sysctrl.xosc32k.modify(|_, w| {
+        #[cfg(not(feature = "samda1"))]
         unsafe {
             // 6 here means: use 64k cycles of OSCULP32k to start up this oscillator
             w.startup().bits(6);
         }
+        #[cfg(feature = "samda1")]
```

Comment:
Interesting. I wonder why the PACs are different here. Maybe the D21 SVD is an older version?